wire-ios-data-model/MLS/CoreCryptoKeyProvider.swift

```diff
@@ -19,6 +19,11 @@
 import Foundation
 
 public class CoreCryptoKeyProvider {
+
+    public init() {
+
+    }
```

Comment:
is this just to make the `init` public ?

Reply:
Yes